packages/components/app/styles/components/tabs.scss

```diff
@@ -77,19 +47,23 @@ $indicator-height: 3px;
       color: var(--token-color-foreground-action);
     }
   }
-}
 
-.hds-tabs__tab-link {
-  @include hds-focus-ring-with-pseudo-element($top: 4px, $right: 6px, $bottom: 4px, $left: 6px);
-  display: block;
-  padding: 8px 12px;
-  color: var(--token-color-foreground-primary);
-  white-space: nowrap;
-  text-decoration: none;
-  border-radius: var(--token-form-control-border-radius);
+  .hds-tabs__tab-link {
+    @include hds-focus-ring-with-pseudo-element($top: 4px, $right: 6px, $bottom: 4px, $left: 6px);
+    display: flex;
+    align-items: center;
+    height: 36px;
+    padding: 8px 12px;
+    color: var(--token-color-foreground-primary);
+    white-space: nowrap;
+    text-decoration: none;
+    border-radius: var(--token-form-control-border-radius);
+
+    > * + * { margin-left: 6px; }
```

Comment:
generally, we tend not to use these generic selectors, but be more specific/declarative. 
or we use a wrapping element with a "contextual" class (eg. in the button https://github.com/hashicorp/design-system/blob/d24ce3e1a19fe6bf85006148af7005ea5be4b5c6/packages/components/addon/components/hds/button/index.hbs#L15-L17) or apply directly the "contextual" class to the icon (eg. in the https://github.com/hashicorp/design-system/blob/d24ce3e1a19fe6bf85006148af7005ea5be4b5c6/packages/components/addon/components/hds/dropdown/list-item/copy-item.hbs#L11-L14)
then we use the specific "contextual" class to apply styles (typically layout).
so in your case would mean add a class `hds-tabs__tab-icon` to the icon and a class `hds-tabs__tab-badge` to the badge, and then do something like:
```
.hds-tabs__tab-icon { margin-right: 6px; }
.hds-tabs__tab-badge { margin-left: 6px; }
```

Reply:
ok